.github/workflows/test-php.yml

```diff
@@ -56,8 +57,23 @@ jobs:
         run: |
           vendor/bin/php-cs-fixer --dry-run --diff fix
           vendor/bin/psalm --no-cache
-          vendor/bin/phpunit
+          vendor/bin/phpunit --testsuite unit
```

Comment:
We can run all the tests in a regular run. The separation between acceptance and unit existed only because we had different methods for testing.